root/static/scripts/release-editor/actions.js

```diff
@@ -69,6 +75,83 @@ const actions = {
     }));
   },
 
+  duplicateReleaseGroups: ko.observableArray([]),
+  loadingDuplicateReleaseGroups: ko.observable(false),
+  failedLoadingDuplicateReleaseGroups: ko.observable(false),
+  duplicateReleaseGroupsRequest: null,
+  duplicateReleaseGroupsQuery: null,
+
+  /**
+   * Query for existing release groups that are credited to any of the
+   * currently-credited artists and have similar names to the release.
+   */
+  loadDuplicateReleaseGroups() {
```

Comment:
This is mostly just moved from `init.js`, but I'll try to point out the minor changes (apart from the calls to the new `failedLoadingDuplicateReleaseGroups` property) in review comments.